src/components/Formik/FormikSelectInput/FormikSelectInput.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SelectInput from '../../SelectInput/SelectInput';
+
+/**
+ * @description Higher order function that provides a closure for the SelectInput onChange handler
+ *  we need to set the values, this is necessary because the Formik Field onChange handler is expecting
+ *  an event object while the SelectInput onChange is only providing the value as the argument
```

Comment:
@nathanyoung I think we generally have a disparity with other components. In TextInput for example, we do pass the event object, which is why this would probably not be needed.

CheckboxInput passes a bool value directly. So we should standardize across the board. For my money I'd rather pass the event directly to the parent, thus always giving the user slightly more responsibility but also a little more flexibility.

Reply:
@nathanyoung @juanfabrega Coming back to this, I also think we should pass the event object. I know this will require some updating on the alchemy side, but I'm happy to see it through.

Reply:
@JCarmine luckily I think they've all used it with Formik so as long as our formik abstraction is there we should be good.